public/index.css

```diff
@@ -4,27 +4,18 @@
 }
 
 :root {
-  --user-font-scale: 1rem - 16px;
-  --max-width: calc(100% - 2rem);
+  --user-font-scale: calc(1em+1vw);
```

Comment:
@natemoo-re can you review these styling changes? Since you originally wrote them I'm not sure what this changes. @aFuzzyBear it would be great if you could add a comment (in this GH thread is fine) mentioning why these changes were needed

Reply:
This wee liner is a bit of typography css trickary i had picked up to make the font-size responsive to the viewport, the em bit is the size of the letters as it bottoms out and the vw is the size of the letters respective to the viewport, It makes the typography alot more fluid and respective to the different viewports alot easier,

https://bits.theorem.co/css-pro-tips-responsive-font-sizes-and-when-to-use-which-units/
https://allthingssmitty.com/2016/12/05/flexible-type-using-root/